clarify how to access value property (basics #3)

terichadbourne opened the pull request:
Closes #209 & #210, addressing feedback received submitted by @frijol and @ajhurliman from the Seattle chapter. I'd love your feedback on whether this addresses your concerns.

fsdiogo commented on src/tutorials/Basics/03.vue:
```diff
@@ -17,13 +17,20 @@ import exercise from './03-exercise.md'
-    return { fail: 'You forgot to return a result :)' }
+    return { fail: 'No result was returned. Perhaps you tried to access the `value` of the object returned by `ipfs.dag.get` without waiting for the promise to be completed (wrapping the `await` statement in parentheses)?' }
   }
+  console.log(result)
```
Remove the logs.

fsdiogo replied:
This one and the lines below.

terichadbourne commented:
Removed the console logs and also realized that I had been giving a variable name to the results of `dag.get` that made it look like it was a CID. Fixed that in the text and solution to refer to the returned data as a decoded block referred to as a node, per @mikeal's suggestion.
